src/sentry/lang/javascript/processor.py

```diff
@@ -300,6 +300,13 @@ def fetch_file(url, project=None, release=None, allow_scraping=True):
 
     Attempts to fetch from the cache.
     """
+    # If our url has been truncated, it'd be impossible to fetch
+    # so we check for this early and bail
+    if url[-3:] == '...':
```

Comment:
This is definitely less than ideal, but there's not really another way to determine this. Alternatively, I first did:

```
if len(url) == 256 and url[-3:] == '...'
```

to try and be more precise, but this will be error prone if we ever changed the length of truncation.

This technically leaves a margin of error. If someone were to have a legit url that ended in 3 periods, we'd fail to fetch now of course, but I don't think that case will realistically happen.